docs/resources/server/recipes/index.md

```diff
@@ -135,7 +142,7 @@ See [the Brewing chapter in the Mob Effects & Potions article][brewing].
 
 ## Custom Recipes
```

Comment:
Since this section has become significantly larger in this update, it might be worth moving it to a separate article.